### ISSUE TITLE
Bug? Cannot find module '../input.json'

### ISSUE THREAD

**Issue by asbjornh** [KyleMayes/eslint-plugin-ordered-imports#2]
It seems like version `0.2.0` of the plugin fails to load. When running eslint with this plugin enabled I get this error:

```
Error: Failed to load plugin 'ordered-imports' declared in '.eslintrc.json': Cannot find module '../input.json'
Require stack:
- ~/node_modules/eslint-plugin-ordered-imports/dist/rule.js
- ~/node_modules/eslint-plugin-ordered-imports/dist/index.js
- ~/node_modules/eslint/lib/cli-engine/config-array-factory.js
- ~/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js
- ~/node_modules/eslint/lib/cli-engine/cli-engine.js
- ~/node_modules/eslint/lib/cli-engine/index.js
- ~/node_modules/eslint/lib/cli.js
- ~/node_modules/eslint/bin/eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (~/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at getSchema (~/node_modules/eslint-plugin-ordered-imports/dist/rule.js:7:18)
    at Object.<anonymous> (~/node_modules/eslint-plugin-ordered-imports/dist/rule.js:16:17)
    at Module._compile (~/node_modules/v8-compile-cache/v8-compile-cache.js:194:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
```

I'll have a look and see if I can do something about it

**Comment by KyleMayes** [comment]
Fix released as `v0.2.1`.

**Comment by asbjornh** [comment]
Great! It seems to work now.
Edit: It works really well!
